core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -1124,6 +1124,17 @@ public ServerBuilder tlsAllowUnsafeCiphers(boolean tlsAllowUnsafeCiphers) {
         return this;
     }
 
+    /**
+     * Returns a {@link ContextPathServicesBuilder} which binds {@link HttpService}s under the
+     * specified context paths.
+     *
+     * @see ContextPathServicesBuilder
+     */
+    @UnstableApi
+    public ContextPathServicesBuilder<ServerBuilder> contextPath(String... contextPaths) {
```

Comment:
Global comment:
Could you also add the variant of this method that takes `Iterable<String>`?